geniza/corpus/management/commands/add_links.py

```diff
@@ -226,7 +242,8 @@ def get_india_book(self, title):
     # base url target for each supported link type
     base_url = {
         "goitein_note": "https://commons.princeton.edu/media/geniza/",
-        "indexcard": "https://geniza.princeton.edu/indexcards/index.php?a=card&id=",
+        "indexcard": "https://geniza.princeton.edu/indexcards/"
+        + quote("index.php?a=card&id="),
```

Comment:
these args should not be quoted
```suggestion
        "indexcard": "https://geniza.princeton.edu/indexcards/index.php?a=card&id=",
```